R/roman_population.R

```diff
@@ -0,0 +1,16 @@
+#' Roman Population Sizes dataset
+#'
+#' The population sizes in the Roman Empire has been a topic of heated debates among some academics.
+#' Various accounts diverge in how they estimate population size and, therefore,
+#' the conclusions about the Roman population at different points in time (see Scheidel2008).
+#' This dataset contains the high and low estimations for the population of mainland
+#' Italy during the Roman Empire (scheidel2008, p. 31).
+#' @source Scheidel, W. (2008). Roman population size: the logic of the debate.
+#' People, land, and politics: demographic developments and the transformation of Roman Italy, 300, 17-70.
```

Comment:
Why did you choose to replace `battles` with `roman_population`? It only uses the range annotation?